src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/inference/CpxVariantCanonicalRepresentation.java

```diff
@@ -348,10 +362,33 @@ static SimpleInterval getAffectedReferenceRegion(final List<SimpleInterval> even
         return new SimpleInterval(eventPrimaryChromosomeSegmentingLocations.get(0).getContig(), start, end);
     }
 
+    // TODO: 3/8/18 to-be-removed in final commit
+    /**
+     * REVIEW COMMENTS:
+     * I think it might be nice to extract this into a method called produceCpxVariantContext(Tuple2<>...).
+     * Then it would be great to write a little unit test demonstrating the functionality (ie getting one variant context back with the right annotations set).
+     *
+     * What if you made this return a VariantContextBuilder instead of a VariantContext, then you wouldn't have to make it,
+     * then make a new builder from the new VariantContext, and then make that?
+     *
+     * REPLY:
+     * As suggested.
+     */
     @VisibleForTesting
     VariantContextBuilder toVariantContext(final ReferenceMultiSource reference) throws IOException {
 
         final CpxVariantType cpxVariant = new CpxVariantType(affectedRefRegion, typeSpecificExtraAttributes());
+
+        // TODO: 3/8/18 to-be-removed in final commit
+        /**
+         * REVIEW COMMENT:
+         * Are we sure we want the end to be equal to the start for these?
+         *
+         * REPLY:
+         * Not sure I understand the comment.
+         * the variable "pos" below is going to be the POS column of the VCF record,
+         * and several lines down below is where we populate the END INFO field.
+         */
         final SimpleInterval pos = new SimpleInterval(affectedRefRegion.getContig(), affectedRefRegion.getStart(), affectedRefRegion.getStart());
```

Comment:
Oh I see. I was confused about what this interval is being used for, sorry. If the variable was named 'refAlleleInterval' or something maybe it would help?